### PR TITLE
fixed issue related not getting the proper geometry onto sql2gee

### PR DESCRIPTION
**Pull request by aagm**

### CODE REVIEW

File: adapterearthengine/services/query_service.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ def convert(query, query_type='sql'):
         endpoint = 'fs2SQL'
 
     result = endpoint+query
+    logging.info(result)
```
Comment (tiagojsag) [inline adapterearthengine/services/query_service.py]: add a small string before the actual value, so it's easy to identify, in the logs, that a given log line was produced by this `logging.info` call

File: adapterearthengine/routes/api/v1/earth_engine_router.py [hunk 2]
```diff
@@ -17,11 +17,14 @@
 def build_query(rq):
     dataset = rq.get_json().get('dataset').get('data')
     sql = rq.args.get('sql', None) or rq.get_json().get('sql', None)
+    logging.debug(f'[ROUTER build_query]: {str(rq.args)}')
```
Comment (tiagojsag) [inline adapterearthengine/routes/api/v1/earth_engine_router.py]: perhaps use `info` instead here?

File: adapterearthengine/services/query_service.py (hunk 1 above)
Comment (aagm) [reply adapterearthengine/services/query_service.py]: Changed as requested on the next commit: https://github.com/resource-watch/adapter-earth-engine/pull/12/commits/ef6191631dfaf8c6c50d1356d83dd2bd129b3462

File: adapterearthengine/routes/api/v1/earth_engine_router.py (hunk 2 above)
Comment (aagm) [reply adapterearthengine/routes/api/v1/earth_engine_router.py]: Changed as requested on the next commit: https://github.com/resource-watch/adapter-earth-engine/pull/12/commits/ef6191631dfaf8c6c50d1356d83dd2bd129b3462

**Pull request merged**